Makefile

```diff
@@ -34,27 +38,47 @@ generate_json:
 
 .PHONY: test
 test: build
-	rm ./out/output.html || true
-	rm ./out/output2.html || true
-	.${BINARY} -name "KVM Linux" -repo "github.com/kubernetes/minikube/" -pr "6096" -in "testdata/minikube-logs.json" -out_html "./out/output.html" -out_summary out/output_summary.json -details "0c07e808219403a7241ee5a0fc6a85a897594339"
-	.${BINARY} -name "KVM Linux" -repo "github.com/kubernetes/minikube/" -pr "6096" -in "testdata/Docker_Linux.json" -out_html "./out/output2.html" -out_summary out/output2_summary.json -details "0c07e808219403a7241ee5a0fc6a85a897594339"
-	.${BINARY} -name "KVM Linux" -repo "github.com/kubernetes/minikube/" -pr "6096" -in "testdata/Docker_Linux.json" -out_html "./out/output2NoSummary.html" -details "0c07e808219403a7241ee5a0fc6a85a897594339"
+	rm -f ./out/output.html
+	rm -f ./out/output2.html
+	rm -f ./out/output2NoSummary.html
+	rm -f ./out/output_summary.json
+	rm -f ./out/output2_summary.json
```

Comment:
It would be cleaner if there was a make command that deleted all theses.

Something like:
```
clean-output
 rm -f ...
```

Then you could have these other command call it:
```
test: clean-output build
...

testdb: clean-output build
...
```

Would remove a lot of duplicated lines

Could instead use `clean` instead as well if there's nothing else in the `out` dir we care about.